cli/src/pcluster/config/imagebuilder_config.py

```diff
@@ -79,13 +79,38 @@ def __init__(self, regions: str, launch_permission: str = None):
         self.launch_permission = Resource.init_param(launch_permission)
 
 
+class AdditionalIamPolicy(Resource):
```

Comment:
Is it possible to reuse and put in common the AdditionalIamPolicy from cluster_config? So that we can also leverage the same validator. (minor - the AdditionalIamPolicyValidator should be moved under the iam_validators package)

Reply:
For now, AdditionalIamPolicyValidator is designed particularly for cluster iam policy, not for image. So I didn't reuse the AdditionalIamPolicy from cluster_config

Reply:
ok, if we cannot use `AdditionalIamPolicyValidator` we need another validator. BTW I think that `AdditionalIamPolicyValidator` needs to be a generic validator that can be used by both cluster_config and imagebuilder_config